app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @welcome_message %></h1>
+<h3><%= @description %></h3>
+<%= form_with url: '/', method: :post, local: true do |f| %>
```

Comment:
could we use root_path instead of '/' here?

Reply:
THANK YOU for pointing that out. Meant to change it... obvious brain fart. Closing this PR to combine with another with some added changes. :)